language_model/rnn_train.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import sqlite3
+import paddle.v2 as paddle
+import ptb
+import gzip
+
+
+def rnn_net(dict_dim, is_generating=False):
```

Comment:
yes it can run when import ptb which is implemented by @pkuyym . It's just training for language model, and  I will implement how to generate later.